assets/configuration/keybindings.json

```diff
@@ -1,9 +1,10 @@
 {
     "bindings": [
+        { "key": "<S-C-P>", "command": "quickOpen.open", "when": ["editorTextFocus"] },
```

Comment:
These bindings - the `<S-C-P>` for quick open, and `<C-P>` for command palette seem reversed to me (compared both to Oni1 or VSCode)

Reply:
yeah they are that's a hang over from me just trying to get it working initially, I'll tweak that